src/main/archetype/ui.tests/test-module/specs/aem/forms.js

```diff
@@ -237,23 +237,38 @@ describe('AEM Forms OOTB Content Tests', () => {
             communication: '#guideContainer-rootPanel-communication___guide-item-nav',
             confirmation: '#guideContainer-rootPanel___guide-item-nav-container > li[title="Confirmation"]'
         };
+        const footer = '.guidefooter';
+        const header = '.guideheader';
         themes.forEach((theme) => {
+            //For each theme take screenshot of all the panels
+            //For those panels whose content does not fit in single screenshot take two screenshots
             it(theme, function () {
                 const verifyPanel = verifyThemePanel.bind(this, theme);
                 browser.url(getThemePath(theme));
-                verifyPanel('Basic info');
+                verifyPanel('Basic info 1');
```

Comment:
Can we have a meaningful name ? Why do we need to use numbers as suffix ?

Reply:
A single screenshot does not capture the panel completely, so we took 2 screenshots for the basic info panel. basic info 1 for the upper part, basic info 2 for the next part.

Reply:
Wrote it in the comment also.

Reply:
Sorry, can we name it as per their semantics ?

Reply:
What about Basic Info Panel and Basic Info Submit Button?